packages/lazy-import/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "@wordpress/lazy-import",
+	"version": "1.0.0-prerelease",
+	"description": "Lazily import a module, installing it automatically if missing.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/lazy-import/README.md",
```

Comment:
Regarding my previous comment about the npm version, it might be a good idea to include `engines` field.

Reply:
> Regarding my previous comment about the npm version, it might be a good idea to include `engines` field.

Added `engines` field in e67a81f0c3.